src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -73,6 +79,11 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
         Metrics.getInstance().getNumericalMetric(MetricName.REQ_COUNT_TOTAL).increment();
         SqlRequest sqlRequest = SqlRequest.NULL;
         try {
+
+            if (!isSQLFeatureEnabled()) {
+                throw new SQLFeatureDisabledException("Either opendistro.sql.enabled or rest.action.multi.allow_explicit_index setting is false");
```

Comment:
np: seems like longer than 120 characters, please break the line if it is. We will eventually enable checkstyle to catch these errors, but for now let's at least not create more.

Reply:
Will do.